db/migrate/20200218042742_create_user_stocks.rb

```diff
@@ -1,6 +1,8 @@
 class CreateUserStocks < ActiveRecord::Migration[6.0]
   def change
     create_table :user_stocks do |t|
+      t.integer :stock_id
```

Comment:
Forgot to add attributes to this migration. If you've already setup your database with `rake db:setup` and migrated this file, you might want to rollback once `rake db:rollback` and migrate `rake db:migrate` once this is merged